docs/usage/configuration-options.md

```diff
@@ -443,6 +443,8 @@ You can configure this to `true` if you prefer Renovate to close an existing Dep
 
 Configure this option if you prefer a different title for the Dependency Dashboard.
 
+## dependencyDashboardLabels
+
```

Comment:
I think you mean to add text here on how to use the feature? 😉

Reply:
If I'm not mistaken, that is automatically generated from the definitions?

https://github.com/renovatebot/renovate/pull/10601/files#diff-5a0608bb8d9d5dbdb218a0169373ad3514241002fba8ca5299507a2b32b87c9cR422-R428

Reply:
I think you're right! 😄